Creando los objetos de la pagina Summer_academy

YaArias opened the pull request:
### Issues

_¿Qué se está resolviendo?_

El overview de la pagina Summer_Academy

### Todos

_¿Qué queda por hacer?_

Crear el css y ver que se ajuste a todo tipo de pantallas.

### Pasos para reproducir

_¿Qué necesitamos hacer para probar?_

Revisar la pagina y correrlo.

YaArias committed:
Hemos creado la pagina html

Aqui hemos creado la pagina html y lo hemos llamado en el archivo
`Summer_academy.html`.

YaArias committed:
Terminacion de los objetos de la pagina overview

Aqui hemos terminado los objetos de la pagina overview y estos se
encuentran tanto en universitarios como en pre-universitarios. Los
objetos se ajustan a cualquier tipo de pantalla.

YaArias committed:
Agregando terminacion de la pagina overview.

Aqui se le ha hecho alguna modificacion a los objetos de la pagina overview.

misscs commented on app/modules/overview/media_object.css:
```diff
@@ -0,0 +1,21 @@
+width: 300px;
+margin: 20px 0 0 20px;
+text-align: center;
+}
```
alineación 🤞 revisa el archivo porque hay problemas en outras líneas

YaArias replied:
@misscs No entiendo cuando dice que hay prolemas en otras lineas !!
me podria aclarar por favor

misscs commented on app/modules/overview/media_object.macro.html:
```diff
@@ -0,0 +1,14 @@
+  <div class="media_object">
+     <div class="media_img">
+      {% if data.image %}
+        <img src = '{{ data.image }}' />
```
Hay dos problemas en la línea 5. qué son?

YaArias replied:
@misscs  oooh , tenemos que pegar el signo de igualdad a src y necesitamos utilizar las comillas dobles.

misscs commented on app/modules/overview/media_object.macro.html:
```diff
@@ -0,0 +1,14 @@
+    <div class="media_content">
+      <h2>{{ data.headline }}</h2>
+      <p class="media_date">{{ data.description }}</p>
+      <h1> {{ data.intro }}</h1>
```
Me puedes explicar tu racionamiento por las etiquetas que esta usando en L9-L11.

YaArias replied:
@misscs 
en la L11 según el plano presentado antes de estar los objetos de la pagina esta debia tener una descripción de la pagina pero me di cuenta que ninguna de estas lineas realizan ninguna función útil.
til.

misscs commented on app/modules/overview/media_object.css:
```diff
@@ -0,0 +1,21 @@
+.media_object {
+float: left;
+height: 400px;
```
No es advisable definir una altura fija. Hay momentos específicos, como tamaño de imagenes, pero cuando estamos trabajando con contenido siempre dejamos que el contenido determina la altura. 

Por eso debes usar flex en vez de float ;)

YaArias replied:
ok. Gracias !!

misscs commented on app/modules/overview/media_object.macro.html:
```diff
@@ -0,0 +1,14 @@
+        <img src = '{{ data.image }}' />
+      {% endif %}
+    </div>
+    <div class="media_content">
```
Cuál es nuestra patron para nombrar classes?

YaArias replied:
@misscs seguir con una secuencia de nombre y que vayan acorde con lo que estamos trabajando !!

misscs reviewed (changes requested):
Algunas preguntas y cambios! 

No olvides revisar el trabajo antes de submitter. Específicamente debes asegurarte que el código cumple a nuestro estilo

YaArias committed:
Realizando cambios en los documentos de los objetos de la pagina
de overview.

Work in process !!

YaArias committed:
Indentacion en las declaraciones de css.

YaArias committed:
Cambio de rayitas de bajo a guion a las clases.

YaArias committed:
Cambio del nombre de la carpeta.

Overview a media_object

YaArias committed:
Poniendo el contenido de la caja flexible.

YaArias committed:
Colocando los objetos dentro de un contenedor.

Hemos creado seccion y hemos agregado los objetos dentro de la misma.

YaArias committed:
Cambiando de raya de abajo por guion.

se ha cambiado de una raya de abajo por un guion en el nombre de el
documento css.

YaArias committed:
Creando el un archivo de css en la carpeta programs.

Se ha agregado un archivo css en la carpeta programs.

YaArias committed:
Cambiando el nombre de el archivo css

YaArias committed:
Agregando el nuevo archivo a style.css

misscs commented on app/modules/media_object/media-object.css:
```diff
@@ -0,0 +1,19 @@
+
```
No empezamos los archivos con un linea

misscs commented on app/modules/media_object/media_object.css:
```diff
@@ -0,0 +1,24 @@
+
```
Mismo comentario

misscs commented on app/views/programs/programs-overview.css:
```diff
@@ -0,0 +1,6 @@
+
+
```
🙀

misscs commented:
Algunos cambios estilísticos. Después de standup vamos a trabajar juntas para resolver los conflictos. 

💯 💃 👯 😁

YaArias committed:
Quitando los espacios en blanco de el archivo.

Hemos quitado los espacios en blanco que hay en el archivo.

YaArias committed:
Archivo borrado.

Se ha eliminado el archivo porque se agregaron los declaraciones a
otro documento individual.

misscs committed:
Merge branch 'master' into overview

 Conflicts:
	app/assets/styles/site.css
	app/views/prep_academy/index.html
	app/views/summer_academy/index.html
	package-lock.json

misscs committed:
Ajustar contenido
